bin/nvh

```diff
@@ -658,20 +691,35 @@ function install() {
     fi
   fi
 
+  if [[ "${NVH_USE_XZ}" = true ]] && [[ "S{g_xz_force_enabled}" = "false" ]]; then
+    can_use_xz || NVH_USE_XZ="false"
+  fi
+
   echo
   install_log "installing" "${g_mirror_folder_name}/${resolved_version}"
 
+  if [[ "${NVH_USE_XZ}" = "true" ]] && [[ "${g_xz_force_enabled}" = "false" ]]; then
+    local resolved_major_version
+    resolved_major_version="$(echo ${resolved_version#v} | cut -d '.' -f '1')"
+    if [[ "${resolved_major_version}" -lt 4 ]]; then
+      NVH_USE_XZ="false"
```

Comment:
Just a thought: the Node version check could be moved back into `can_use_xz()`.

Not sure which I prefer, but putting it in `can_use_xz()` could be a bit more DRY, due to not having to check for `NVH_USE_XZ` again.

Could gate any of the checks in `can_use_xz()` (e.g. just the platform checks) with: `if [[ "${g_xz_force_enabled}" = "false" ]]; then` if need be.